crates/primitives/src/revm/env.rs

```diff
@@ -156,7 +156,9 @@ pub fn fill_tx_env_with_beacon_root_contract_call(env: &mut Env, parent_beacon_b
             source_hash: None,
             mint: None,
             is_system_transaction: Some(false),
-            enveloped_tx: None,
+            // The L1 fee is not charged for the EIP-4788 transaction, submit zero bytes for the
+            // enveloped tx size.
+            enveloped_tx: Some(Bytes::default()),
         },
```

Comment:
@mattsse @Rjected confirming this is OK?

Reply:
yeah this should be fine, i think we eventually want to deprecate the `enveloped_tx` field and replace it with
```rust
struct L1TxFeeInfo {
    zeros: u32,
    ones: u32
}
```
or something